docs/manage_infrastructure.md

```diff
@@ -473,30 +527,27 @@ When you click on "Install all patches", XOA will do all of the following automa
 - upload them
 - apply them in the correct order
 
-You can see more screenshots here: https://xen-orchestra.com/blog/hotfix-xs70e004-for-xenserver-7-0/
+You can see [more screenshots here](https://xen-orchestra.com/blog/hotfix-xs70e004-for-xenserver-7-0).
 
 :::tip
 If you are behind a proxy, please update your `xo-server` configuration to add a proxy server, as [explained in the appropriate section](configuration.md#proxy-for-xenserver-updates-and-patches).
 :::
 
-### Notes on patching
+#### Host updates
 
-- Xen Orchestra won't reboot your hosts automatically. That's your call to choose when to do it.
-- Patching doesn't always require rebooting. Check in the host view if the reboot warning is displayed, it means you need to reboot to have the patch fully applied (see screenshot below)
-- XO will install all patches without rebooting: that's not an issue. Even applying patches manually, **it's not mandatory to reboot after each patch**.
-
-![](./assets/xo5patching.png)
+As for XCP-ng, we do NOT recommend to install updates to individual hosts. Please install patches for the whole pool, even if it's a single host.
 
 ## Pool Management
 
-::: danger
+:::danger
 As specified in the [documentation](https://xcp-ng.org/docs/requirements.html#pool-requirements) your pool shouldn't consist of hosts from different CPU vendors.
 :::
 
-::: warning
-- Even with matching CPU vendors, in the case of different CPU models XCP-ng will scale the pool CPU ability to the CPU having the least instructions.
+:::warning
+
+- Even with matching CPU vendors, in the case of different CPU models, XCP-ng/Citrix Hypervisor will "level" down to use the CPU having the least instructions.
 - All the hosts in a pool must run the same XCP-ng version.
-:::
+  :::
```

Comment:
Fucking linter, how do they work.

Reply:
Our linter does not work with Vuepress special syntax, you need to bypass it when committing `-n`.

Reply:
Thanks, I'll try to remember it. Or maybe it's documented somewhere? Ideally, could we tell the linter to STFU for all files in the doc folder?

Reply:
That might be the best thing to do indeed.